app/Test.js

```diff
@@ -0,0 +1,106 @@
+let express = require("express");
+let { Pool } = require("pg");
+let bcrypt = require("bcrypt");
+let env = require("../env.json");
+
+let hostname = "localhost";
+let port = 3000;
+let app = express();
+
+app.use(express.json());
+app.use(express.static("public"));
+
+let pool = new Pool(env);
+pool.connect().then(() => {
+    console.log("Connected to database");
+});
+
+// https://github.com/kelektiv/node.bcrypt.js#a-note-on-rounds
+let saltRounds = 10;
+
+app.post("/signup", (req, res) => {
+    let username = req.body.username;
+    let plaintextPassword = req.body.plaintextPassword;
+
```

Comment:
add a check
"select username from users where username = username"
This should give 0 rows since every username should be unique